dela.sh

```diff
@@ -0,0 +1,38 @@
+source .env;
```

Comment:
I'd also suggest adding a shebang and possibly a `bash -e` to ensure errors stop the script.
The script doesn't seem robust to failures.

Reply:
as Ahmed said, hopefully we can soon remove the script altogether it is mainly for our conveniance at the moment

nonetheless, I added the the shebang and removed the source